EM_FHPath.py

```diff
@@ -259,7 +259,7 @@ def execute(self, obj):
             self.ww = [Vector(0,0,0)]
         else:
             # transform 'obj.ww' according to the 'Base' Placement
-            # (transation is don't care, we worry about rotation)
+            # (translation is don't care, we worry about rotation)

```

Comment:
is this correct or should it be `transaction` ?

Reply:
correct thx